scripts/generate-statics-package-json.js

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+const fs = require('fs-extra');
+const path = require('path');
+const { oneAppStaticsDir } = require('./utils');
+const { version, contributors, license } = require('../package.json');
+
+const pathToStaticsPackageJson = path.resolve(oneAppStaticsDir, 'package.json');
+
+const doWork = async () => {
```

Comment:
maybe rename to writeStaticPackageJson

Reply:
I wanted to keep the very descriptive name `doWork()` for consistency with the other scripts 😅. I will change it.